### ISSUE TITLE
Plugin install error

### ISSUE THREAD

**Issue by edro** [lrills/helm-unittest#17]
When run install command

```sh
$ helm plugin install https://github.com/lrills/helm-unittest
Error: Unable to update checked out version
```
and when run this:

```sh
$ helm plugin install https://github.com/lrills/helm-unittest --version v0.1.0
/home/-----/.helm/plugins/helm-unittest/install-binary.sh: line 14: type: cygpath: not found
Downloading https://github.com/lrills/helm-unittest/releases/download/v0.1.0/helm-unittest-linux-0.1.0.tgz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   620    0   620    0     0    746      0 --:--:-- --:--:-- --:--:--   746
100 2893k  100 2893k    0     0   750k      0  0:00:03  0:00:03 --:--:-- 1359k
Preparing to install into /home/-----/.helm/plugins/helm-unittest
cp: cannot stat '/tmp/helm-unittest/tpl*': No such file or directory
Failed to install helm-unittest
\tFor support, go to https://github.com/kubernetes/helm.
Error: plugin install hook for "unittest" exited with error
```
OS and helm versions:
```sh
$ lsb_release -a
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.4 LTS
Release:	16.04
Codename:	xenial

$ helm version
Client: &version.Version{SemVer:"v2.8.2", GitCommit:"a80231648a1473929271764b920a8e346f6de844", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.8.2", GitCommit:"a80231648a1473929271764b920a8e346f6de844", GitTreeState:"clean"}
```

**Comment by lrills** [comment]
My bad, fixed in v0.1.1 now.
Please try again!

**Comment by lrills** [comment]
`Error: Unable to update checked out version` is traced in #18
